libsql-ffi/build.rs

```diff
@@ -124,127 +124,6 @@ pub fn build_bundled(out_dir: &str, out_path: &Path) {
         .flag("-D_POSIX_THREAD_SAFE_FUNCTIONS") // cross compile with MinGW
         .warnings(false);
 
-    if cfg!(feature = "wasmtime-bindings") {
```

Comment:
This was a mistake. Let me know if you want this PR merged and I will re-add them.

Reply:
We do want to merge the PR so please re-add them